install_config/configuring_openstack.adoc

```diff
@@ -15,6 +15,13 @@ When deployed on link:https://www.openstack.org/[OpenStack], {product-title} can
 be configured to access OpenStack infrastructure, including
 xref:../install_config/persistent_storage/persistent_storage_cinder.adoc#install-config-persistent-storage-persistent-storage-cinder[using OpenStack Cinder volumes as persistent storage] for application data.
 
+[[configuring-opestack-permissions]]
+== Permissions
+Configuring OpenStack for {product-title} requires following permissions:
+
+* *`_member_`* - For creating assets, you need the member role on the project.
```

Comment:
Do we outline anything about LBaaS? And what needs to be installed for router ingress?